photon/index.html

```diff
@@ -0,0 +1,526 @@
+              <nav class="navbar navbar-expand-xl ">
+                  <div class="collapse navbar-collapse" >
+                      <ul class="navbar-nav mr-auto" style="color: white;">
+                          <li class="nav-item"  >
+                              <a class="nav-link navbar-brand">
+                                  <i class="fa fa-twitter fa-lg " ></i>
+                              </a>
+                          </li>
+                          <li class="nav-item">
+                              <a class="nav-link navbar-brand ">
+                                  <i class="fa fa-facebook fa-lg "></i>
+                              </a>
+                          </li>
+                          <li class="nav-item">
+                              <a class="nav-link  ">
+                                  <i class="fa fa-instagram fa-lg" ></i>
+                              </a>
+                          </li>
+                      </ul>
+                  </div>
+              </nav>
+          </div>
+          
+          <div class="col logo-col">
+              <img src="https://jevelin.shufflehound.com/portfolio1/wp-content/uploads/sites/12/2017/03/Logo_w.png">
+          </div>
+           <div class="col">
+              <i class="fa fa-bars fa-lg" style="color: white;"></i>
+          </div>
+      </div>
+
+      <div class="row">
+          <div class="col-12 header-title ">
+              <p >Alex <strong>Photon</strong></p>
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-6 offset-3 ">
+              <p class="header-p">
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse dignissim consectetur feugiat. Fusce quis ex fringilla, facilisis velit et, euismod mauris. Vivamus quis consectetur nisi. Ut a est enim. Suspendisse imperdiet nisl id <a href="#">metus posuere mattis.</a>
+              </p> 
+          </div>
+      </div>
+      <div class="row bottom-nav">
+          <div class="col-3"></div>
+          <div class="col-6 ">
+              <span class="badge badge-pill filter-item selected">All</span>
+              <span class="badge badge-pill filter-item">Art</span>
+              <span class="badge badge-pill filter-item">Commerical</span>
+              <span class="badge badge-pill filter-item">Design</span>
+              <span class="badge badge-pill filter-item">Music</span>
+              <span class="badge badge-pill filter-item">Mock-up</span>
+              <span class="badge badge-pill filter-item">Photography</span>
+              <span class="badge badge-pill filter-item">Tech</span>
+          </div>
+          <div class="col-3"></div>
+      </div>
+  </div>
+</header>
+<!-- end shan section -->
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<!-- ahmed's section -->
+
+<!-- image gallery  -->
+<div class="container">
+    <div class="card-columns">
+      <div class="card" >
+          <div class="img-hover-effect">
```

Comment:
Awesome work on laying the cards into 3 columns and implementing the card hover effect!
It looks fantastic and visually pleasing! I love it! 👌